src/main/java/io/vertx/ext/stomp/Stomp.java

```diff
@@ -0,0 +1,43 @@
+package io.vertx.ext.stomp;
+
+import io.vertx.codegen.annotations.VertxGen;
+import io.vertx.core.Vertx;
+import io.vertx.core.net.NetServer;
+
+/**
+ * Interface used to create STOMP server and clients.
+ *
+ * @author <a href="http://escoffier.me">Clement Escoffier</a>
+ * @see StompClient
+ * @see StompServer
+ */
+@VertxGen
+public interface Stomp {
+
+  String UTF_8 = "utf-8";
+
```

Comment:
Ah.. I notice that these methods are also on StompServer... any reason to have them in two places?